example/src/App.tsx

```diff
@@ -5,28 +5,27 @@
  * @format
  */
 
-import React from 'react';
+import type React from 'react';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { FastImageTab } from './FastImageTab';
 import { NitroImageTab } from './NitroImageTab';
 import { createStaticNavigation } from '@react-navigation/native';
 import { EmptyTab } from './EmptyTab';
+import { NitroAssetImageTab } from './NitroAssetImageTab';
 
 const Tabs = createBottomTabNavigator({
-  detachInactiveScreens: false,
-  screens: {
-    Empty: EmptyTab,
-    FastImage: FastImageTab,
-    NitroImage: NitroImageTab,
-  },
+	detachInactiveScreens: false,
```

Comment:
it seems like you have applied some different formatting - this is 8 spaces (or 2 tabs?) instead of what I had before - 2 spaces.